net/ssl/ssl_key_logger_impl.cc

```diff
@@ -48,7 +49,7 @@ class SSLKeyLoggerImpl::Core {
     if (!file_)
       return;
 #if defined(STARBOARD)
-    file_->WriteAll(line.c_str(), line.length());
+    starboard_api::FileWriteAll(file_->file(), line.c_str(), line.length());
```

Comment:
Nit: does this even need to go direct to Starboard, or can it just use base::File::WriteAtCurrentPos() which routes to FileWriteAll anyway ?